CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ All notable changes to the "vscode-didact" extension will be documented in this
  - Adding title-level context menu item to Markdown and AsciiDoc editors to call validation
  - Adding awareness so Didact view column is persisted if moved
  - Adding new 'vscode.didact.copyToClipboardCommand' command that can put text on the clipboard
+ - Improving validation to show number of commands validated and open Didact output channel to see results
+ - Improving command validation code to use find vs. filter 
```

Comment:
not sure it is worthy to mention that, this is internal. users doesn't really care. Or from a user point of view, it would be interesting to have "improve performance for discovering available commands" or something like that

Reply:
I think if we made the change, we should note it even if the user doesn't care. Most users don't look very closely at changelog anyway.

Reply:
what is interesting is that the performance are improved. I think this is what needs to be mentioned.